docs/userguide/modules/getting-started/pages/installing-service-binding.adoc

```diff
@@ -59,3 +61,180 @@ image:sbo_install_options.png[sbo_install_options]
+----
+
+Remove --namespace and --create-namespace flag if you wish to install the chart on default namespace.
+
+As part of Helm test we delete the deploymemt,service binding resource and secret created for testing the operator.
+
+Optional: To view the  resources created for testing, install the chart with the `keepTestResources` flag value set to `true`:
+[source,bash]
+----
+helm install service-binding-operator-release \
+service-binding-operator-helm-chart/service-binding-operator \
+--namespace service-binding-operator --create-namespace \
+--set keepTestResources=true
+----
+
+.Example output
+[source,terminal]
+NAME: service-binding-operator-release
+LAST DEPLOYED: Mon May 16 09:15:16 2022
+NAMESPACE: service-binding-operator
+STATUS: deployed
+REVISION: 1
+
+You can check whether the chart is succesfully installed by running the following command
+
+[source,bash]
+----
+kubectl get pods --namespace service-binding-operator
+----
+
+[## Testing the  chart]
+== Testing the chart 
+In order to test the chart the user is expected to create a secret (specify the namespace if applicable), named my-k-config from his kubeconfig .
+
+NOTE:
+In case you are installing the chart on AWS eks cluster then you need to modify the aws-auth configmap.
+----
+kubectl edit -n kube-system cm/aws-auth 
+----
+. Add `-system:masters` to mapRoles and save.
+. After editing the config map, update the eks kubeconfig:
+----
+aws eks update-kubeconfig --name <cluster-name>
+----
+
+
+Then Continue with the following steps.
+
+----
+kubectl create secret generic my-k-config --from-file=kubeconfig=<PATH TO YOUR KUBECONFIG> --namespace service-binding-operator
+----
+
+.Example output
+[source,terminal]
+----
+secret/my-k-config created
+----
+
+The previous output verifies that the `my-k-config` secret is created.
+
+Run the Helm test (specify the namespace if applicable) using :
+
+```
+helm test service-binding-operator-release --namespace service-binding-operator
+```
+.Example output
+[source,terminal]
+----
+NAME: service-binding-operator-release
+LAST DEPLOYED: Mon May 16 10:44:53 2022
+NAMESPACE: service-binding-operator
+STATUS: deployed
+REVISION: 1
+TEST SUITE:     service-binding-operator-release-test
+Last Started:   Mon May 16 11:01:10 2022
+Last Completed: Mon May 16 11:01:22 2022
+Phase:          Succeeded
+----
+The `Succeeded` phase from the output indicates that the Helm test has run successfully. 
+
+Verify that the Helm test has run successfully:
+
+[source,bash]
+----
+kubectl get pods --namespace service-binding-operator
+----
+
+.Example output
+[source,terminal]
+----
+NAME                                              READY   STATUS      RESTARTS   AGE
+service-binding-operator-release-test             0/1     Completed   0          4m28s
+----
+
+This implies that you have successfully installed the service binding operator using a Helm chart and are able to bind your workload to backing services.
```

Comment:
The output verifies that you have successfully installed the {servicebinding-title} using a Helm chart and are able to bind your workload to backing services.